lua/buffertabs/init.lua

```diff
@@ -134,20 +144,19 @@ local function create_win(name, is_active, is_modified, data_idx)
 
     -- setup buffer
     local buf = api.nvim_create_buf(false, true)
-    local modified_icon = is_modified and cfg.modified or ""
-    modified_icon = #cfg.modified == 0 and "" or modified_icon -- if you dont want icon
+    local modified = is_modified and cfg.modified or ""
```

Comment:
As far as I can tell, `modified_icon` does not exist so this may be behaviour that has since been removed? Either way it wasn't being used so I simplified